config/settings.d/dns_nsupdate.yml.example

```diff
@@ -0,0 +1,16 @@
+---
+#
+# Configuration file for 'nsupdate' dns provider
+#
+# Can be true, false
+:enabled: true
```

Comment:
I mean users need to do two steps to configure a provider, what happens when they setup disabled one? We should cover this with tests. I'd prefer to simplify it a bit more effectively getting rid of the dns.yml.